.github/workflows/pipeline.yml

```diff
@@ -33,7 +33,7 @@ jobs:
             path: bandit_security.json
             retention-days: 5
         - name: Run performance tests
-          run: python -m locust --users 3 --spawn-rate 1 --run-time 1s --headless
+          run: python -m locust --users 3 --spawn-rate 1 --run-time 15s --headless
```

Comment:
Is this change intentional?

Reply:
@tferns Yes was intentional as I had to increase the runtime to accommodate the new microservice for the performance tests.

Reply:
@tferns It's only for this repo, the other repos already have different runtimes which are more than the one here.